tests/test_image_types.py

```diff
@@ -152,7 +152,11 @@ def test_src_set_with_format(self):
         self.assertIn("width-100.format-webp.webp", data["srcSet"])
         self.assertIn("width-300.format-webp.webp", data["srcSet"])
 
-    def test_src_set_invalid_format(self):
+    @skipIf(
+        WAGTAIL_VERSION >= (5, 2),
+        "Wagtail 5.2 introduced changes to this error message.",
+    )
+    def test_src_set_invalid_format_wagtail_below_5_2(self):
```

Comment:
Please add a docstring detailing this test, and it's relevance with the other test.

Reply:
Thanks, docstring added.